pkg/restore/restore.go

```diff
@@ -834,13 +839,13 @@ func (ctx *restoreContext) shouldRestore(name string, pvClient client.Dynamic) (
 
 // crdAvailable waits for a CRD to be available for use before letting the
 // restore continue.
-func (ctx *restoreContext) crdAvailable(name string, crdClient client.Dynamic) (bool, error) {
+func (ctx *restoreContext) crdAvailable(name string, crdClient client.Dynamic, resourceTimeout time.Duration) (bool, error) {
 	crdLogger := ctx.log.WithField("crdName", name)
 
 	var available bool
 	// Wait 1 minute rather than the standard resource timeout, since each CRD
 	// will transition fairly quickly.
```

Comment:
I think changing the timeout to the resourceTimeout should be safe, but also need to remove the comments here.